everyone-on-support.md

```diff
@@ -0,0 +1,33 @@
+# Prologue
```

Comment:
I would use Overview or Introduction, to keep with the standard wording of Handbook.